examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLExample.scala

```diff
@@ -60,7 +60,8 @@ object SparkSQLExample {
 
   private def runBasicDataFrameExample(spark: SparkSession): Unit = {
     // $example on:create_df$
-    val df = spark.read.json("examples/src/main/resources/people.json")
+    val df = spark.read.json(
+      "file:///" + System.getProperty("user.dir") + "/examples/src/main/resources/people.json")
```

Comment:
Please check `fs.default.name`. If the path does not specify the file system in the schema, then, I believe the property will be referred. If it is not found, it will use `file:///`[1].

[1]https://github.com/apache/hadoop/blob/7e521c5a49fbcf88285c102051ea2522edc847b9/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/CommonConfigurationKeysPublic.java#L58